src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -40,6 +40,10 @@ constexpr const auto UpdatePatternLocationsInterval = std::chrono::milliseconds(
 
 DEFINE_ENUM_FLAG_OPERATORS(winrt::Microsoft::Terminal::TerminalControl::CopyFormat);
 
+// This is usually defined in wpc.h, but we don't need that. Surprisingly, there
+// isn't a better HRESULT version of ERROR_FILE_NOT_FOUND
+#define E_FILE_NOT_FOUND 0x80070002

```

Comment:
TODO: Mike, this isn't `E_FILE_NOT_FOUND`. 


0x80070003 | -2147024893 | E_PATH_NOT_FOUND
-- | -- | --
| Hex | Dec | As Win32 error code '0x3' |
| 0x3 | 3 | ERROR_PATH_NOT_FOUND

Reply:
Why not just write it as `HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND)` or 

`#define E_FILE_NOT_FOUND HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND)`?

Reply:
prefer the HR_FROM_WIN32, yeah

Reply:
🤦‍♂️ I spent a bunch of time trying to get `HRESULT_FROM_NTSTATUS` to work right. `NTSTATUS`. That's not the right function. guh.